### PR TITLE
Add CI scripts to test the tutorial

### PR DESCRIPTION
**Pull request by damienmg**

### CODE REVIEW

**Comment by damienmg** [comment]
Can you review it? I git hub is not really nice but meh

File: tutorial/ci/WORKSPACE [hunk 1]
```diff
@@ -0,0 +1,6 @@
+android_sdk_repository(
+    name = "androidsdk",
+    path = "/usr/local/google/home/dmarting/bin/android-sdk-linux",
```
Comment (kchodorow) [inline tutorial/ci/WORKSPACE]: Does this work?

File: tutorial/ci/build.sh [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+if [ -n "${ANDROID_SDK_PATH-}" ]; then
+  "${BAZEL}" build //android
+fi
+
+# Under darwin, test the ios application
+if [ "$(uname -s | tr 'A-Z' 'a-z')" = "darwin" ]; then
+  "${BAZEL}" build //ios-app --ios_sdk_version=${IOS_SDK_VERSION:-8.4}
+fi
+
+echo "Yay!"
```
Comment (kchodorow) [inline tutorial/ci/build.sh]: Add a link to the ci in the tutorial/README.md.

File: tutorial/ci/WORKSPACE (hunk 1 above)
Comment (damienmg) [inline tutorial/ci/WORKSPACE]: Arf that's shouldn't be there sorry, reverting

File: tutorial/ci/build.sh (hunk 2 above)
Comment (damienmg) [inline tutorial/ci/build.sh]: Done

**Pull request merged**